ios/app/brave_core_switches.h

```diff
@@ -0,0 +1,81 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_APP_BRAVE_CORE_SWITCHES_H_
+#define BRAVE_IOS_APP_BRAVE_CORE_SWITCHES_H_
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NSString* BraveCoreSwitchKey NS_STRING_ENUM;
+/// Overrides the component updater source. Defaults to the CI provided value
+///
+/// Expected value: url-source={url}
+OBJC_EXPORT const BraveCoreSwitchKey BraveCoreSwitchKeyComponentUpdater;
+/// Overrides Chromium VLOG verbosity. Defaults to only printing from folders
+/// existing within a `brave` subfolder up to level 5.
+///
+/// Expected value: {folder-expression}={level}
+OBJC_EXPORT const BraveCoreSwitchKey BraveCoreSwitchKeyVModule;
+/// Overrides the sync service base URL. Defaults to the CI provided value
+///
+/// Expected value: A URL string
+OBJC_EXPORT const BraveCoreSwitchKey BraveCoreSwitchKeySyncURL;
+/// Sets a number of overrides for the ads & ledger services such as which
+/// environment its using, debug mode, etc.
+///
+/// Expected value: A comma-separated list of flags, including:
+///     - staging={bool}
+////    - development={bool}
+///     - debug={bool}
+///     - reconcile-interval={int}
+///     - retry-interval={int}
+OBJC_EXPORT const BraveCoreSwitchKey BraveCoreSwitchKeyRewardsFlags;
+/// Overrides the number of seconds to upload P3A metrics
+///
+/// Expected value: A number (in seconds)
+OBJC_EXPORT const BraveCoreSwitchKey
+    BraveCoreSwitchKeyP3AUploadIntervalSeconds NS_SWIFT_NAME(p3aUploadIntervalSeconds);  // NOLINT
```

Comment:
The formatter places the `NS_SWIFT_NAME` annotations on the end of the line regardless of line length for some reason